designs/osc_containerization/OSC-containerization.md

```diff
@@ -86,6 +88,49 @@ N/A
 ### OSC Synchronization Tasks
 N/A
 
+## OSC Kolla Container  
+In order for OSC to be available in the [TripleO docker repository](https://hub.docker.com/u/tripleoupstream/) it must build as part of the OpenStack Kolla.  The main requirement for this is the presence of a Docker file template `Dockerfile.j2` for OSC upstreamed to the [Kolla docker folder](https://github.com/openstack/kolla/tree/master/docker).  Below is the initial intended OSC Docker template file to be reviewed with the Kolla team:
```

Comment:
I am not sure whether we will have just the Dockerfile.j2. I was thinking that this will have sub-services like OSC-base and the OSC, similar to other kolla services.